languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -24575,7 +24575,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example correction="einander">Unsere Sinne beeinflussen <marker>sich gegenseitig</marker>.</example>
                 <example correction="einander">Freunde sollten <marker>sich gegenseitig</marker> helfen.</example>
             </rule>
-            <rule default="temp_off">
+            <rule>
```

Comment:
Da ist ein FP [im Diff](https://internal1.languagetool.org/regression-tests/via-http/2022-03-28/de-DE_full/result_grammar_SICH_GEGENSEITIG[2].html) – bitte aufpassen wie ein Schießhund!

Reply:
@udomai ich verstehe nicht ganz, warum das ein FP ist. Die Regel schlägt an, wie sie soll, oder?

'Beispiele für Strukturen, in denen zwei Funktionen **sich gegenseitig zueinander** distributiv verhalten, sind Boolesche Algebren, wie die Algebra der Mengen oder die Schaltalgebra.'

Hier wird 'gegenseitig' rausgelöscht und dann heißt der Satz:

'Beispiele für Strukturen, in denen zwei Funktionen **sich zueinander** distributiv verhalten, sind Boolesche Algebren, wie die Algebra der Mengen oder die Schaltalgebra.'

Ansonsten wäre es doppelt gemoppelt. Das war doch der Kontext, in dem die SICH_GEGENSEITIG[1] öfter ausgeschaltet wurde.

Reply:
Ah, my bad!